cylc/flow/scripts/cat_log.py

```diff
@@ -261,18 +261,18 @@ def view_log(
         # Print location even if the workflow does not exist yet.
         print(logpath)
         return 0
-    if not os.path.exists(logpath) and batchview_cmd is None:
```

Comment:
The `logpath` contains the filename (defaults to `job.out`) even if it was not specified, even if we are listing a directory.

Consequently this caused failures when listing remote directories where the `job.out` file was not present.